app/components/post_list/post/body/reactions/reactions.tsx

```diff
@@ -66,7 +67,7 @@ const Reactions = ({currentUserId, canAddReaction, canRemoveReaction, disabled,
     const [sortedReactions, setSortedReactions] = useState(new Set(reactions.map((r) => getEmojiFirstAlias(r.emojiName))));
     const styles = getStyleSheet(theme);
 
-    useEffect(() => {
+    useDidUpdate(() => {
```

Comment:
This was changing the state at first render, executing the ordering with the values it already had, re-rendering all its children. Using useUpdate we save ~30ms and one less render.